SimCalorimetry/HGCalSimProducers/interface/HGCHEbackDigitizer.h

```diff
@@ -3,6 +3,42 @@
 
 #include "SimCalorimetry/HGCalSimProducers/interface/HGCDigitizerBase.h"
 #include "DataFormats/HGCDigi/interface/HGCDigiCollections.h"
+#include "DataFormats/ForwardDetId/interface/HGCScintillatorDetId.h"
+
+
+class HGCHEbackSignalScaler
```

Comment:
if this information eventually needs to be accessed in multiple places (e.g. also local reco), this class should be promoted to an `ESSource`, but probably not necessary right now